pkg/k8s/k8s_service.go

```diff
@@ -28,9 +28,9 @@ func GetService(namespace, name string) (*v1.Service, error) {
 }
 
 // DeployService deploys a service if it does not exist.
-func DeployService(namespace, name string, labels, selectorMap map[string]string, portsTCP []int, portsUDP []int) (*v1.Service, error) {
+func DeployService(namespace, name string, labels, annotations, selectorMap map[string]string, portsTCP []int, portsUDP []int, serviceType v1.ServiceType) (*v1.Service, error) {
```

Comment:
as this function is growing, what do you think if for a better readability we change the style to:

```go
func DeployService(
	namespace, 
	name string, 
	labels, 
	annotations, 
	selectorMap map[string]string, 
	portsTCP []int, 
	portsUDP []int, 
	serviceType v1.ServiceType) 
(*v1.Service, error) { 
```

Reply:
https://github.com/celestiaorg/knuu/pull/152/commits/e371faab7c9f662bef4d180d9ad496c4a8a6a60f